src/tools/kata-ctl/src/args.rs

```diff
@@ -39,6 +39,9 @@ pub enum Commands {
 
     /// Display version details
     Version,
+
+    /// Set Timeout value
+    Timeout(TimeoutArguments),
```

Comment:
The timeout member needs to be part of `KataCtlCli` rather than here as we want to command line to parse like this:

```bash
$ kata-ctl --timeout 1234 exec ...
```

Not like this:

```bash
$ kata-ctl timeout 1234
```

Reply:
Sure, I will try it

Reply:
I added timeout as option in kata ctl cli. Now it can be used as follow:

```kata-ctl --timeout <num> exec ...```